README.md

```diff
@@ -1,106 +1,78 @@
 
-This app comes with Celery.
+### poetry
 
-To run a celery worker:
+install `poetry` https://poetry.eustace.io/docs/
 
-``` bash
-cd scoap3
-celery -A config.celery_app worker -l info
+```bash
+$ curl -sSL https://install.python-poetry.org | python3 - --version 1.1.14
 ```
 
-Please note: For Celery's import magic to work, it is important *where* the celery commands are run. If you are in the same folder with *manage.py*, you should be right.
+### pre-commit
 
-To run [periodic tasks](https://docs.celeryq.dev/en/stable/userguide/periodic-tasks.html), you'll need to start the celery beat scheduler service. You can start it as a standalone process:
+install `pre-commit` https://pre-commit.com/
 
-``` bash
-cd scoap3
-celery -A config.celery_app beat
+```bash
+$ curl https://pre-commit.com/install-local.py | python -
 ```
 
-or you can embed the beat service inside a worker with the `-B` option (not recommended for production use):
+And run
 
-``` bash
-cd scoap3
-celery -A config.celery_app worker -B -l info
+```bash
+$ pre-commit install
 ```
 
-### Email Server
-
-In development, it is often nice to be able to see emails that are being sent from your application. For that reason local SMTP server [MailHog](https://github.com/mailhog/MailHog) with a web interface is available as docker container.
-
-Container mailhog will start automatically when you will run all docker containers.
-Please check [cookiecutter-django Docker documentation](http://cookiecutter-django.readthedocs.io/en/latest/deployment-with-docker.html) for more details how to start all containers.
+### Docker & Docker Compose
 
-With MailHog running, to view messages that are sent by your application, open your browser and go to `http://127.0.0.1:8025`
+Follow the guide https://docs.docker.com/compose/install/
 
-### Sentry
+---
 
-Sentry is an error logging aggregator service. You can sign up for a free account at <https://sentry.io/signup/?code=cookiecutter> or download and host it yourself.
-The system is set up with reasonable defaults, including 404 logging and integration with the WSGI application.
+## Run with docker
 
-You must set the DSN url in production.
-
-## Deployment
+### Start
+```bash
+$ docker-compose up
+```
 
-The following details how to deploy this application.
+### Usage
 
-### Docker
+After startup, the application should be avaiable at [localhost:3000](localhost:3000)
 
-See detailed [cookiecutter-django Docker documentation](http://cookiecutter-django.readthedocs.io/en/latest/deployment-with-docker.html).
+---
 
-### Custom Bootstrap Compilation
+## How to test
 
-The generated CSS is set up with automatic Bootstrap recompilation with variables of your choice.
-Bootstrap v5 is installed using npm and customised by tweaking your variables in `static/sass/custom_bootstrap_vars`.
+### Backend
+To run the backend tests locally, first start the application using docker-compose and then use:
```

Comment:
Basically, you'll probably need db, mq, es and redis. Maybe we can split docs about testing into local and docker parts (I think it's similar on inspire)?